README.md

```
@@ -1,2 +1,75 @@
+- On Ubuntu:
+```bash
+sudo apt-get install libconfig++-dev
+sudo apt-get install libsfml-dev
+```
+- On MacOs:
+```bash
+brew install libconfig
+brew install sfml@2 
+```
+
+### 2. Compile the project with CMake 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)